pex/compatibility.py

```diff
@@ -100,32 +108,33 @@ def nested(*context_managers):
 
 
 else:
-    from contextlib import nested
+    from contextlib import nested as nested
 
 
 if PY3:
     import urllib.parse as urlparse
 else:
-    import urlparse
+    import urlparse as urlparse
 
 
 if PY3:
-    from queue import Queue
+    from queue import Queue as Queue
 
     # The `os.sched_getaffinity` function appears to be supported on Linux but not OSX.
     if not hasattr(os, "sched_getaffinity"):
-        from os import cpu_count
+        from os import cpu_count as cpu_count
     else:
 
         def cpu_count():
+            # type: () -> Optional[int]
```

Comment:
Technically, this could be `-> int`, but MyPy requires that the function signature is identical to the `if not hasattr` branch.

Lmk if I should comment this.

Reply:
I think this is fine, that should be expected for anything in an `if` block like this.

Reply:
Cool. And I figure an eager-beaver like me would try deleting it, then MyPy will complain and they'll realize why it's needed hah.